svg-2-android-vector/settings.gradle

```diff
@@ -0,0 +1 @@
+rootProject.name = "svg2androidvector"
```

Comment:
Are these files necessary? I'm not too familiar with it but would prefer to avoid unnecessary files if possible.

Reply:
They're strongly recommended: https://docs.gradle.org/current/userguide/organizing_gradle_projects.html#always_define_a_settings_file

Reply:
I've read through the documentation and believe the `settings.gradle` is recommended only for the root folder because you could rename the directory containing the project.  In [part 1 of this guide](https://docs.gradle.org/current/userguide/multi_project_builds.html#naming_recommendations) Gradle recommends against renaming subprojects so they remain consistent with the directory names, though given the current structure it would be awkward to come up with a good root project name without renaming the `svg-2-android-vector` folder to `plugin` or something like that.

Reply:
I understand now why you needed this (though I couldn't get it to build locally for me), but it should be called `svg-2-android-vector` to be consistent with the folder name.

Reply:
Split out into #86 and applied your suggestion (I think I had originally done that, but removed the dashes to try to debug why it wasn't loading for me). no hurt feelings if it is closed without merge since you have a system that works for you 👍